.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: Report an issue with DXC
+title: ''
+labels: ['bug', 'needs-triage']
+assignees: ''
+
+---
+
+**Description**
+<!--- Please provide a few sentences describing the issue you encountered. --->
+
+**Steps to Reproduce**
+<!--- Provide a description of how to reproduce the error. If possible please 
+provide source and tool command line options. If the issue reproduces on
```

Comment:
Might be worth asking for preprocessed source and specifiying how to generate it (e.g. `/P`) since a preprocessed source can be easier to work with than a zip full of shader/header files.